tools/teal/dkey/dsign/main.go

```diff
@@ -62,8 +62,13 @@ func main() {
 		ddata, err := ioutil.ReadFile(os.Args[3])
 		failFast(err)
 
+		programstr := fmt.Sprintf("%s", pdata)
```

Comment:
Why using `fmt.Sprintf` where `fmt.Sprint` would do the work (notice no `f` in function name)?

Reply:
I'm using Sprint because I've got a byte[] containing a string (the code) and I want to create a string. The alternative is to use:
programstr := string(pdata)
 
Your proposed change doesn't work because it converts the byte[] to a string containing the binary codes (e.g.: '27 34 93 59') instead of a string with the TEAL code.